Description: Relax dependencies for Debian
 Some of the build dependencies are only needed for Windows and macOS, let's
 drop them entirely as we don't need them for Debian. Also, change the
 `futures-lite` (used for autopkgtests) to a version compatible with Debian
 packages.
Author: Arnaud Ferraris <aferraris@debian.org>
Forwarded: not-needed
Last-Update: 2024-11-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: a/Cargo.toml
===================================================================
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -87,10 +87,10 @@
 version = "0.4.9"
 
 [dev-dependencies.env_logger]
-version = "0.10.0"
+version = "0.11.0"
 
 [dev-dependencies.futures-lite]
-version = "1.13.0"
+version = "2"
 
 [target.'cfg(any(target_os="linux", target_os="android"))'.dependencies.libc]
 version = "0.2.155"
@@ -103,29 +103,6 @@
     "net",
 ]
 
-[target.'cfg(target_os="macos")'.dependencies.core-foundation]
-version = "0.9.3"
-
-[target.'cfg(target_os="macos")'.dependencies.core-foundation-sys]
-version = "0.8.4"
-
-[target.'cfg(target_os="macos")'.dependencies.io-kit-sys]
-version = "0.4.0"
-
-[target.'cfg(target_os="windows")'.dependencies.windows-sys]
-version = "0.48.0"
-features = [
-    "Win32_Devices_Usb",
-    "Win32_Devices_DeviceAndDriverInstallation",
-    "Win32_Foundation",
-    "Win32_Devices_Properties",
-    "Win32_Storage_FileSystem",
-    "Win32_Security",
-    "Win32_System_IO",
-    "Win32_System_Registry",
-    "Win32_System_Com",
-]
-
 [lints.rust.unexpected_cfgs]
 level = "warn"
 priority = 0
